Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -72,10 +76,23 @@ HRESULT __DWriteGetFontListForFamilyName(CFStringRef familyName, IDWriteFontList
     BOOL exists = FALSE;
 
     RETURN_IF_FAILED(fontCollection->FindFamilyName(reinterpret_cast<wchar_t*>(unicharFamilyName.data()), &index, &exists));
+    if (exists) {
+        ComPtr<IDWriteFontFamily> fontFamily;
+        RETURN_IF_FAILED(fontCollection->GetFontFamily(index, &fontFamily));
+
+        RETURN_IF_FAILED(
+            fontFamily->GetMatchingFonts(DWRITE_FONT_WEIGHT_NORMAL, DWRITE_FONT_STRETCH_NORMAL, DWRITE_FONT_STYLE_NORMAL, outFontList));
+
+        return S_OK;
+    }
+
+    RETURN_HR_IF(E_INVALIDARG, !__UserCreatedFontCollection);
```

Comment:
This failure represents trying to create a font with an invalid font family name, and will fail the same whether or not the user registered any custom fonts #ByDesign